includes/class-wporg-notifications.php

```diff
@@ -402,17 +359,9 @@ private static function get_project_to_translate( WP_Comment $comment ): GP_Proj
 	private static function get_main_projects():array {
 		global $wpdb;
 
-		$main_projects = $wpdb->get_results(
-			$wpdb->prepare(
-				"
-			SELECT id
-			FROM {$wpdb->gp_projects}
-			WHERE parent_project_id IS NULL"
-			),
-			ARRAY_N
-		);
+		$main_projects = $wpdb->get_col( "SELECT id FROM {$wpdb->gp_projects} WHERE parent_project_id IS NULL" );
```

Comment:
@amieiro A few comments here:
- `get_results` can be replaced with `get_col` here
- `prepare` is only necessary when you need to insert external variables.